### PR TITLE
Messages.page

### PR DESCRIPTION
**Pull request by igmonk**

### CODE REVIEW

File: src/components/ContactList/styles.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default ({ mainColorPalette }) => ({
+  root: {
+    height: '100%',
+    borderRight: '2px solid',
+    borderRightColor: mainColorPalette[100]
```
Comment (siarhei-sia) [inline src/components/ContactList/styles.js]: Please, remove it. Write borderRight: \`2px solid ${mainColorPalette[100]}\`

File: src/components/MessageHistoryItem/styles.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    height: 'auto',
+    padding: '6px'
+    // Material UI workaround end.
+  },
+  message: {
+    display: 'inline-flex',
+    paddingLeft: '10px',
+    paddingRight: '10px',
+    paddingTop: '5px',
+    paddingBottom: '5px',
```
Comment (siarhei-sia) [inline src/components/MessageHistoryItem/styles.js]: Please, replace it with padding: '5px 10px 5px 10px
'

File: src/components/MessageHistory/styles.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export default ({ mainColorPalette }) => ({
+  root: {
+    height: '100%',
+    borderBottom: '2px solid',
+    borderBottomColor: mainColorPalette[100],
```
Comment (siarhei-sia) [inline src/components/MessageHistory/styles.js]: Do the same : remove `borderBottomColor`, please

**Review by siarhei-sia — APPROVED** [review]

**Comment by igmonk** [comment]
@sergey-moiseyenko pls check once again

**Pull request merged**